src/components/CalendarMonthGrid.jsx

```diff
@@ -33,6 +35,7 @@ const propTypes = forbidExtraProps({
   onMonthTransitionEnd: PropTypes.func,
   renderDay: PropTypes.func,
   transformValue: PropTypes.string,
+  daySize: PropTypes.number,
```

Comment:
Since this propType definition appears in multiple places, let's use a shared one so the source of truth only lives in one place.